mmcv/runner/hooks/logger/wandb.py

```diff
@@ -39,6 +43,9 @@ def before_run(self, runner):
             self.wandb.init(**self.init_kwargs)
         else:
             self.wandb.init()
+        
+        if self.config_path is not None:
+            os.system(f"cp -r {self.config_path} {self.wandb.run.dir}")
```

Comment:
The following lines also work in the Windows environment.
```python
import shutil
shutuil.copy2(self.config_path, self.wandb.run.dir)
```

Reply:
Okay. I thank you for recommendation.

Reply:
It is changed in under commit.